app/api-v1/services/recipeService.js

```diff
@@ -21,7 +21,18 @@ async function getRecipes() {
   return getRecipesDb()
 }
 
+async function getRecipeByID(id) {
+  const recipeResult = await getRecipeByIDdb(id)
+  if (recipeResult.length === 0) {
+    return { statusCode: 404, result: {} }
+  } else {
+    const result = recipeResult[0]
+    return { statusCode: 200, result }
+  }
+}
+
```

Comment:
I think we should throw the `ItemNotFoundError` here as that's more consistent with the pattern elsewhere